docs/changelog.md

```diff
@@ -419,6 +419,7 @@ This starts a chat session:
 ```
 Type 'exit' or 'quit' to exit
 Type '!multi' to enter multiple lines, then '!end' to finish
+Type '!edit' to open your default editor and modify the prompt.
```

Comment:
I'm going to drop this from the changelog as I'm OK with that showing how the feature looked back in version 0.10.